lib/client/signalflow/signalflow_client.js

```diff
@@ -73,7 +73,67 @@ function SignalflowClient(apiToken, options) {
       }
     };
   }
+/**
+ * This is a method on  SignalflowClient that registers a live tail and exposes the stream and close methos to
+ * get live tail data and to stop a live tail.
+ * @param opts - {
+ *                query: {
+ *                  matcher: {
+ *                    params: {
+ *                      op: string,
+ *                      args: object
+ *                    },
+ *                  }
+ *                },
+ *                throttleOptions: { rate: number }
+ *              }
+*/
```

Comment:
The indentation of that comment doesn't seem to match that of the function

Reply:
fixed